vita3k/gui/include/gui/state.h

```diff
@@ -121,6 +126,11 @@ struct GuiState {
     size_t memory_editor_start = 0;
     size_t memory_editor_count = 0;
 
+    bool isButtonClicked = false;
+    bool isCapturingKeys = false;
+    int oldCapturedKey = 0;
+    int capturedKey = 0;
```

Comment:
We like to keep our names consistent where we can too, if you can rename stuff to `the_underscore_style` that would be a big help.
```suggestion
    bool is_button_clicked = false;
    bool is_capturing_keys = false;
    int old_captured_key = 0;
    int captured_key = 0;
```